spec/rails_helper.rb

```diff
@@ -61,4 +61,8 @@
     ActiveFedora::Cleaner.clean!
   end
 
+  config.after(:each) do
+    ActiveFedora::Cleaner.clean!
```

Comment:
This should not be necessary since we're using transactional fixtures, and it might even cause some future tests to fail unexpectedly (e.g., that use `before(:all)` to persist records).  Can you clean the database selectively in the tests that need it?